pkgs/development/libraries/qscintilla/default.nix

```diff
@@ -33,7 +33,7 @@ stdenv.mkDerivation rec {
   # Make sure that libqscintilla2.so is available in $out/lib since it is expected
   # by some packages such as sqlitebrowser
   postFixup = ''
-    ln -s $out/lib/libqscintilla2_qt5.so $out/lib/libqscintilla2.so
+    ln -s $out/lib/libqscintilla2_qt5${stdenv.hostPlatform.extensions.sharedLibrary} $out/lib/libqscintilla2${stdenv.hostPlatform.extensions.sharedLibrary}
```

Comment:
Could we make this a bit more readable by having something like:

```nix
let
  libExt = stdenv.hostPlatform.extensions.sharedLibrary;
in
```

at the top of the file?